src/test/regress/expected/replicate_reference_tables_to_coordinator.out

```diff
@@ -19,11 +19,7 @@ SELECT create_reference_table('squares');
 INSERT INTO squares SELECT i, i * i FROM generate_series(1, 10) i;
 NOTICE:  executing the copy locally for shard xxxxx
 CREATE INDEX CONCURRENTLY squares_a_idx ON squares (a);
-SELECT substring(current_Setting('server_version'), '\d+')::int > 11 AS server_version_above_eleven
```

Comment:
interesting trick, we could consider using such pattern if needed for PG15 support, instead of adding a new output file